aiida_quantumespresso/parsers/pw.py

```diff
@@ -149,11 +158,12 @@ def parse_with_retrieved(self, retrieved):
             # We compute here the transpose and its inverse of the structure cell basis, which is needed to transform
             # the parsed rotation matrices, which are in crystal coordinates, to cartesian coordinates, which are the
             # matrices that are returned by the _get_qe_symmetry_list staticmethod
-            cell = structure_data['cell']['lattice_vectors']
-            cell_T = numpy.transpose(cell)
-            cell_Tinv = numpy.linalg.inv(cell_T)
 
             try:
+                cell = structure_data['cell']['lattice_vectors']
```

Comment:
Why is this placed inside the try block? The only case I can see this triggering is if `structure_data['cell']` does not have the element `lattice_vectors` (given that the presence of `cell` is guaranteed by the conditional of this block) and I am not sure if this can ever happen. In any case, if this can happen, it should be in its own try-except block, because if this triggers an `KeyError` a misleading error will be logged, related to missing symmetries.

Reply:
Before I made the changes in `raw_parser_pw.parse_raw_output`, the `structure_data` was empty if the xml file was not found, hence this was my initial quick fix. But yes it should not need to be in there now (or at least in its own try-except block)